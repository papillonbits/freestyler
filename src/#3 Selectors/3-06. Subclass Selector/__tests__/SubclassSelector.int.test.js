import { regular } from './SubclassSelector.int.story'

describe('<SubclassSelector />', () => {
  describe('Render', () => {
    test('must match regular()', () => {
      expect(global.renderToJSON(regular())).toMatchSnapshot()
    })
  })
})
