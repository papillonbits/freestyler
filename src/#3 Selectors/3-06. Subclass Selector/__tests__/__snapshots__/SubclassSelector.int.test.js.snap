// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<SubclassSelector /> Render must match regular() 1`] = `
<div>
  <div
    class="subclass-selector"
  >
    <h1>
      Subclass Selector
    </h1>
    <div>
      <p
        class="button square"
      >
        Square
      </p>
      <p
        class="button rounded"
      >
        Rounded
      </p>
      <p
        class="button go"
      >
        Go
      </p>
    </div>
  </div>
</div>
`;
